README.md

```diff
@@ -207,11 +209,15 @@ If using a version of ImageProcessor.Web version [4.5.0](https://www.nuget.org/p
         <setting key="MaxBytes" value="8194304"/>
         <setting key="Timeout" value="30000"/>
         <setting key="Host" value="http://[myAccountName].blob.core.windows.net/media"/>
+	<!-- new since version 4.5.? -->
+	<setting key="SASQueryString" value="?sv=2018-03-28&amp;si=media-r&amp;sr=c&amp;sig=drXDwEyrzyOKP3QX3e1puVzURvCJMU1oYpBk5oKsrZY%3D"/>
       </settings>
     </service>
   </services>  
 </security>
 ```
+***Note 3*** _SASQueryString_ can easily be configured via **_Microsoft Azure Storage Explorer_**. 
+Read documentation for SAS: https://docs.microsoft.com/en-us/azure/storage/common/storage-dotnet-shared-access-signature-part-1
```

Comment:
I've actually added a specific `AzureImageService` to the `AzureBlobCache` (unpublished so far) so any changes would take place there.

Reply:
I have yes, I'll need to finish some planned work there though. There's some improvements I need to do.